mlrun/db/base.py

```diff
@@ -555,7 +555,7 @@ def get_model_endpoint(
         end: Optional[str] = None,
         metrics: Optional[list[str]] = None,
         features: bool = False,
-    ):
+    ) -> "mlrun.model_monitoring.model_endpoint.ModelEndpoint":
```

Comment:
any reason to keep the `"`?
```suggestion
    ) -> mlrun.model_monitoring.model_endpoint.ModelEndpoint:
```

Reply:
cd4344f